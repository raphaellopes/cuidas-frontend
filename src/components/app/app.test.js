// vendors
import React from 'react';
import renderer from 'react-test-renderer';

// locals
import App from './index';

jest.mock('redux-saga', () => () => ({ run: jest.fn() }));

const component = renderer.create(
  <App>
    My children text
  </App>,
);

describe('<App />', () => {
  test('Should render the component', () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
