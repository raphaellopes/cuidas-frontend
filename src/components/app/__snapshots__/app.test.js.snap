// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Should render the component 1`] = `
<div
  className="sc-bwzfXH ZoqEH"
>
  <header
    className="sc-htpNat dplynj"
  >
    <img
      alt="Cuidas"
      className="sc-bdVaJa katCQl"
      src="logo.svg"
    />
  </header>
  <div>
    <div
      className="sc-ifAKCX fVKAHz"
    >
      <p>
        Selecione a página que deseja visualizar:
      </p>
      <div
        className="sc-EHOje fCSTCe"
      >
        <a
          href="/agendamentos"
          onClick={[Function]}
        >
          <i
            className="icon fa fa-calendar sc-bxivhb kaPDeN"
          />
          Agendamentos
        </a>
        <br />
        <a
          href="/administracao"
          onClick={[Function]}
        >
          <i
            className="icon fa fa-user-md sc-bxivhb kaPDeN"
          />
          Administração
        </a>
      </div>
    </div>
  </div>
</div>
`;
